lua/orgmode/config/mappings/map_entry.lua

```diff
@@ -85,7 +85,16 @@ function MapEntry:attach(default_mapping, user_mapping, opts)
 
   local map_opts = vim.tbl_extend('force', self.opts, opts or {})
 
+  local prefix = ''
+  if map_opts.prefix then
+    prefix = map_opts.prefix
+    map_opts.prefix = nil
```

Comment:
When I pass it via `opts`, I need to remove the key or `vim.keymap` throws an error